src/Widgets/PasswordEditor.vala

```diff
@@ -196,8 +196,7 @@ namespace SwitchboardPlugUserAccounts.Widgets {
                 }
             }
 
-            if (new_pw_entry.get_text () == confirm_pw_entry.get_text ()
-            && new_pw_entry.get_text () != "" && is_obscure) {
+            if (new_pw_entry.get_text () == confirm_pw_entry.get_text () && new_pw_entry.get_text () != "" && is_obscure) {
```

Comment:
Could we use the property `text` here instead of `get_text`? Would make the line length a little better and the code more in line with the rest of the tidying.